src/sentry/profiles/consumers/process/factory.py

```diff
@@ -15,7 +16,7 @@ def process_message(message: Message[KafkaPayload]) -> None:
     sampled = is_sampled(message.payload.headers)
 
     if sampled or options.get("profiling.profile_metrics.unsampled_profiles.enabled"):
-        process_profile_task.delay(payload=message.payload.value, sampled=sampled)
+        process_profile_task.delay(payload=b64encode(message.payload.value), sampled=sampled)
```

Comment:
The return of b64encode() is `bytes`, I think we need `.encode('utf8')` here as well.